imap_processing/ultra/tests/conftest.py

```diff
@@ -0,0 +1,5 @@
+"""Pytest fixtures"""
+
+pytest_plugins = [
```

Comment:
Why do we need the plugins, can't you just put the fixtures directly in this file?